stable/wordpress/templates/secrets.yaml

```diff
@@ -14,9 +14,4 @@ data:
   {{ else }}
   wordpress-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
-  {{ if .Values.databasePassword }}
```

Comment:
@viglesiasce it appears the `databasePassword` field is not used. Looks like it may have been added by mistake. Makes sense to have it removed.